ts/features/payments/checkout/hooks/usePaymentFailureSupportModal.tsx

```diff
@@ -82,8 +92,25 @@ const usePaymentFailureSupportModal = ({
     "";
 
   const zendeskAssistanceLogAndStart = () => {
+    if (!isReady(zendeskPaymentCategory)) {
+      return;
+    }
+    const { payments } = zendeskPaymentCategory.value;
+    const subCategory = getSubCategoryFromFaultCode(payments, faultCodeDetail);
+    // attach the main zendesk category to the ticket
+    addTicketCustomField(
+      zendeskCategoryId,
+      defaultZendeskPaymentCategory.value
+    );
+
+    if (subCategory) {
+      // if a subcategory is found, we attach its id and value to the ticket
+      const { value, zendeskSubCategoryId } = subCategory;
+      addTicketCustomField(zendeskSubCategoryId, value);
+      return;
+    }
     resetCustomFields();
```

Comment:
The `resetCustomFields` must be executed before setting any other ticket custom field (in this case before row n° `100`)